pyensign/ensign.py

```diff
@@ -48,6 +49,18 @@ def __init__(
             Set to True to disable topic ID caching.
         """
 
+        try:
+            with open(cred_path, "r") as file:
+                data = json.load(file)  # Process the loaded JSON data here
```

Comment:
Can we move these comments to above the lines instead of inline?

```python
# Process the loaded JSON data
data = json.load(file)
```

This way it's more consistent with the rest of the comments in the code.